esphome/components/esp32_can/esp32_can.cpp

```diff
@@ -0,0 +1,122 @@
+#include "esp32_can.h"
+#include "esphome/core/log.h"
+
+#include "driver/gpio.h"
+#include "driver/can.h"
```

Comment:
```suggestion
#include <driver/can.h>
```
test this include style

Reply:
This did not seem to work. My guess is that this CI task is missing the nessecary include directories.